web/src/components/atoms/Icon/icons.ts

```diff
@@ -79,6 +82,8 @@ export default {
   listBullets: ListBullets,
   numberNine: NumberNine,
   link: Link,
+  linkSolid: LinkSolid,
+  unlinkSolid: Unlink,
```

Comment:
Maybe it's better to name the second one `UnlinkSolid`.

Reply:
Changed!